drivers/clock_control/clock_control_rpi_pico.c

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright (c) 2022 Andrei-Edward Popa
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT raspberrypi_pico_syscon
+
+#include <zephyr/drivers/clock_control/rpi_pico_clock_control.h>
```

Comment:
This is the only place this header is used (as far as I can see), can it be removed and its contents be copied here?

Reply:
A user can use the structure rpi_pico_clk_setup to setup a clock inside its application